spec/unit/generator/entry_spec.rb

```diff
@@ -538,8 +538,9 @@ def default_sections
 
         subject { described_class.new(options) }
 
-        it "returns 8 sections" do
-          expect(entry_sections.size).to eq 8
+        it "returns 9 sections" do
+          entry_sections.each { |sec| pp(sec.name) }
+          expect(entry_sections.size).to eq 9
```

Comment:
Can you tell when this test began failing?

Has the behavior changed, like "there's a new expected Section"?

Reply:
Yes! And you _did_!